src/Orleans/Streams/Predicates/ExactMatchStreamNamespacePredicate.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Orleans.Streams
+{
+    [Serializable]
+    internal class ExactMatchStreamNamespacePredicate : IStreamNamespacePredicate
```

Comment:
@xiazen Shall I add also `RegexImplicitStreamSubscriptionAttribute` in this case since this predicate requires a parameter to be passed?